packages/tupaia-web/src/features/DashboardItem/DashboardItem.tsx

```diff
@@ -50,17 +50,25 @@ const Title = styled(Typography).attrs({
   line-height: 1.4;
 `;
 
-const getShowDashboardItemTitle = (config: DashboardItemConfig, legacy?: boolean) => {
-  const { presentationOptions, type, viewType, name } = config;
+const getShowDashboardItemTitle = (config?: DashboardItemConfig, legacy?: boolean) => {
+  if (!config) return false;
+  const { type, name } = config;
   if (!name) return false;
-  if (viewType === 'multiValue') {
-    // if report is legacy, show title because it won't have the config set
-    return (
-      (presentationOptions as MultiValueViewConfig['presentationOptions'])?.isTitleVisible || legacy
-    );
+  if (type === 'component') return false;
+  if (type === 'view') {
+    const { viewType } = config;
+    if (
+      viewType === 'multiValue' &&
+      'presentationOptions' in config &&
```

Comment:
It's a little annoying to have to type check for presentations every time they're used, and I did explore a util function but it ended up being not that useful since there are several different shapes of `presentationOptions` so I could never get the types exactly right in a generic function (mainly because if you have a union type and pass a key into a function as keyof the union type, it only gives you the common keys that apply to everything in that union type, not all the possible keys. Seemed more trouble than it was worth to make it work).